README.md

```diff
@@ -25,16 +25,16 @@ The easiest way to get Service Center is to use one of the pre-built release bin
 
 [github-release]: https://github.com/servicecomb/service-center/releases/
 
-### Building and Running Service Center
+### Running Service Center using the Release
 
-You don't need to build from source to use Service Center (binaries on the [GitHub releases page][github-release]).When you get these binaries, you can execute the start script to run Service Center.
+You can download our latest release from [GitHub releases page][github-release]).When you get these release, you can execute the start script to run Service Center.
```

Comment:
We may only provide the source release in the github, let the user download the binary from Apache.